src/core/public/notifications/toasts/global_toast_list.tsx

```diff
@@ -54,8 +72,8 @@ export class GlobalToastList extends React.Component<Props, State> {
     return (
       <EuiGlobalToastList
         data-test-subj="globalToastList"
-        toasts={this.state.toasts}
-        dismissToast={this.props.dismissToast}
+        toasts={this.state.toasts.map(convertToEui)}
+        dismissToast={toast => this.props.dismissToast(toast.id)}
```

Comment:
As we now got our own `Toast` type that's different from EUI, dismiss now also accepts the toast id to avoid falsy type conversion